integrations/purchase/__tests__/bridge-api-categories.test.js

```diff
@@ -0,0 +1,20 @@
+import { getEntryByKey } from '../../../co2eq/purchase';
+import { ALL_ACTIVITY_TYPES } from '../../../definitions';
+import { BRIDGE_API_CATEGORIES } from '../bridge/bridge-api-categories';
+
+const CATEGORY_MAPPINGS = Object.entries(BRIDGE_API_CATEGORIES).filter(
+  ([_key, category]) => category
+);
```

Comment:
Why are you using filter for this? Isn't it just mapping and always returning something? :)

Reply:
`BRIDGE_API_CATEGORIES` returns `null` for all categories that are not mapped.

Reply:
Ah!